src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -288,38 +278,35 @@ def acr_build(cmd,
         try:
             size = os.path.getsize(tar_file_path)
         except OSError:
-            raise CLIError(
-                "Get the size of file {0} failed.".format(tar_file_path))
+            raise CLIError("Get the size of file {0} failed.".format(tar_file_path))
         finally:
             try:
-                logger.debug(
-                    "Starting to delete the archived source code from '%s'.", tar_file_path)
+                logger.debug("Starting to delete the archived source code from '%s'.", tar_file_path)
                 os.remove(tar_file_path)
             except OSError:
                 pass
-        unit = ""
+        unit = 'GiB'
         for S in ['Bytes', 'KiB', 'MiB', 'GiB']:
             if size < 1024:
                 unit = S
                 break
             size = size / 1024.0
-        if unit == "":
-            unit = "GiB"
-        print("Sending build context ({0:.3f} {1}) to ACR".format(size, unit))
+        print("Sending build context ({0:.3f} {1}) to ACR.".format(size, unit))
     else:
-        print("Sending build context to ACR")
+        print("Sending build context to ACR.")
 
     queued_build = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
-        build_request=build_request, resource_group_name=resource_group_name,
-        registry_name=registry_name))
+        resource_group_name=resource_group_name,
+        registry_name=registry_name,
+        build_request=build_request))
 
     if no_logs:
         return queued_build
-    else:
-        build_id = queued_build.build_id
-        print("Queued a build with ID: {0}".format(build_id))
-        print("Waiting for a build agent...")
-        acr_build_show_logs(cmd, client, registry_name, build_id, resource_group_name)
+
+    build_id = queued_build.build_id
+    print("Queued a build with build ID: {}".format(build_id))
+    print("Waiting for a build agent...")
+    return acr_build_show_logs(client, build_id, registry_name, resource_group_name)
```

Comment:
nit: return is not needed here.

Reply:
Just to make pylint happy. We also had return in `build_task.py`.